dd4hep-common.file

```diff
@@ -0,0 +1,14 @@
+%define dd4hep_version v01-19x
+%define tag cc335b34e9eb2825ab18e20c531be813a92d141f
+%define branch master
+%define github_user AIDASoft
+%define keep_archives true
+
+Source: git+https://github.com/%{github_user}/DD4hep.git?obj=%{branch}/%{tag}&export=%{n}-%{realversion}&output=/%{n}-%{realversion}.tgz
+BuildRequires: cmake
+Requires: root boost clhep xerces-c geant4
+
+%prep
```

Comment:
though this section is common but better to move `%prep` section to `dd4hep-geant4.spec` and `dd4hep.spec`. 
Also remove `geant4` dependency from here and add it to `dd4hep-geant4.spec`. The `dd4hep-geant4.spec` should start with 
```
## IMPORT dd4hep-common
### RPM external dd4hep-geant4 %{dd4hep_version}
Requires: geant4

%prep
%setup -n %{n}-%{realversion}
```